ksiren-okhttp3-plugin/src/main/java/com/brightspace/ksiren/okhttp3_plugin/KSirenOkhttp3RequestBuilder.kt

```diff
@@ -29,10 +29,8 @@ class KSirenOkhttp3RequestBuilder(wrappedAction: Action): KSirenRequestBuilder<R
 		if (action.fields.isNotEmpty()) {
 			when (action.method) {
 				"GET" -> {
-					fieldValues.forEach {
-						(name, value) ->
-
-						urlBuilder?.addQueryParameter(name, value)
+					action.fields.forEach {
```

Comment:
Is there a reason to use `action.fields` over `fieldValues`?

Reply:
`fieldValues` was a map of Siren fields owned/serialized by this plugin, which was odd to me, since this should only be concerned with creating a request, SRP and all (I'd also prefer it to not have its own Siren-specific builder pattern, but, trying to keep the upstream changes minimal). This allows Siren-specific behaviour (serialization, validation, such as it is, etc) to be centralized on the Siren classes.